src/screens/HomeScreen.js

```diff
@@ -20,10 +20,10 @@ const HomeScreen = ({ navigation, route }) => {
         <Text style={styles.buttonTextStyle}>New Habit</Text>
```

Comment:
can you add a bit more styling to this text to make it look better on screen, it is too close to the plus button on the image you have sent.